docs/site/Inside-Loopback-Application.md

```diff
@@ -0,0 +1,79 @@
+---
+lang: en
+title: 'Inside a Loopback Application'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Inside-Loopback-Application.html
+---
+
+## The Shopping Cart Example
```

Comment:
Let's set up some context first before we introduce the shopping cart example. It can be something around the following points:

Digital transformation is happening for all business. We need to build open APIs to expose business functions to Web, Mobile and IoT. Developers are looking for productive and easy ways to create APIs from existing data and services.